corehq/sql_db/connections.py

```diff
@@ -147,9 +147,6 @@ def _populate_connection_map(self):
         if UCR_ENGINE_ID not in self.db_connection_map:
             self._add_django_db(UCR_ENGINE_ID, 'default')
 
-        self._add_django_db_from_settings_key(ICDS_UCR_ENGINE_ID, 'ICDS_UCR_DATABASE_ALIAS')
-        self._add_django_db_from_settings_key(ICDS_TEST_UCR_ENGINE_ID, 'ICDS_UCR_TEST_DATABASE_ALIAS')
-
```

Comment:
Is it safe to remove these? Will teh icds dashboard still work? (it's still referenced in the router)